src/Tizen.NUI/src/public/DragAndDrop/DragEvent.cs

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Drag event type.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum DragType
+    {
+      /// <summary>
+      /// The drag object has entered the target view.
+      /// </summary>
+      Enter,
+      /// <summary>
+      /// The drag object has leaved the target view.
+      /// </summary>
+      Leave,
+      /// <summary>
+      /// The drag object moves in the target view.
+      /// </summary>
+      Move,
+      /// <summary>
+      /// The drag object dropped in the target view.
+      /// </summary>
+      Drop
+    }
+
+    /// <summary>
+    /// This specifies drag event.
+    /// </summary>
+    /// Suppress warning : This struct will be used data of callback, so override equals and operator does not necessary.
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1815: Override equals and operator equals on value types")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct DragEvent
+    {
+        private DragType type;
+        private Vector2 position;
+        private string data;
+        /// <summary>
+        /// The drag event type
+        /// </summary>
+        public DragType Type
```

Comment:
Type은 이미 `GetType`이라는 메소드가 있어서 이름이 겹칩니다.
```suggestion
        public DragType DragType
```